index-fr.html

```diff
@@ -42,7 +42,7 @@
     <div class="overlay">
       <div>
         <h1>Le pouvoir multiplicateur des masques</h1>
-        <h3>Un essai interactif sur la manière dont les masques peuvent mettre un terme à la COVID-19</h3>
+        <h3>Un essai interactif sur la manière dont les masques peuvent mettre un terme à COVID-19</h3>
```

Comment:
Same here, it should be either "à la COVID-19" or "au COVID-19" (since "à le" is conflated into "au").